models/user.js

```diff
@@ -0,0 +1,58 @@
+const mongoose = require("mongoose");
+
+const { Schema } = mongoose;
+const ObjectId = Schema.Types.ObjectId;
+
+/**
+ * User Model
+ */
+module.exports = mongoose.model(
+  "User",
+  new Schema({
+    firstName: {
+      type: String,
+      required: true,
+    },
+    lastName: {
+      type: String,
+      required: true,
+    },
+    email: {
+      type: String,
+      required: true,
+      unique: true,
+    },
+    password: {
+      type: String,
+      required: true,
+    },
```

Comment:
Since we are using Firebase for authentication now, I don't think we need to save passwords.